server/middleware/forceHTTPS.js

```diff
@@ -0,0 +1,12 @@
+function forceHTTPS(req, res, next) {
+
+    if (process.env.FORCE_HTTPS === 'true' && !req.secure) {
+       return res.redirect("https://" + req.headers.host + req.url);
```

Comment:
nit: airbnb style guide prefers single quotes over double quotes, and for string concatenation they prefer template strings. Consider doing this:
```
`https://${req.headers.host}${req.url}`
```